test/unit/math/mathOps/include/Defines.hpp

```diff
@@ -0,0 +1,72 @@
+/** Copyright 2019 Jakob Krude, Benjamin Worpitz
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <cmath>
+#include <iostream>
+#include <iomanip>
+#include <limits>
+
+
+namespace alpaka {
+namespace test {
+namespace unit {
+namespace math {
+
+    // New types need to be added to the switch-case in DataGen.hpp
+    enum class Range
+    {
+        ONE_NEIGHBOURHOOD,
+        POSITIVE_ONLY,
+        POSITIVE_AND_ZERO,
+        NOT_ZERO,
+        UNRESTRICTED
+    };
+
+    // New types need to be added to the operator() function in Functor.hpp
+    enum class Arity
+    {
+        UNARY = 1,
+        BINARY = 2
```

Comment:
For consistency with the other `enum`s in alpaka (e.g. `enum class GridBlockExtentSubDivRestrictions`) we should make the values `CamelCase`.

Reply:
also for `enum class Range`